### DISCUSSION TITLE
beach multi choice discussion (WIP)

### DISCUSSION THREAD

**Discussion by samuelhackwill** [samuelhackwill/le-tiret-du-six#47] [Show and tell]
yo, feel free to have a look at this WIP of the discussion with le gérant de la plage. (it's an interactive PDF)

[la plage 27102021.pdf](https://github.com/samuelhackwill/le-tiret-du-six/files/7427566/la.plage.27102021.pdf)

**Comment by JoBalcony** [comment]
ola
I think I read the whole text, nice.
Comments on the form :
> there is at least one bug in the links from answers to correspondant text : "2.En realite je suis agent immobilier >>16" brings you to 17.
> the pds double column is not very practical to navigate but I guess that isn't really an issue for latter.

Comments on the content 
> it's a funny moment kind of coming out of the blue, it looks like an excuse to give your funny impressions on society and the world.
> It's quite far from computers not to say it has absolutely nothing to do with them.
> I feel some subjects you touch, specifically about the youth being arab or jew seem to me that they are too vast of a subject to be just hinted at so quickly.
> I find the description of the mime sentence disproportionately long.
> other than that, it's mostly a good walk, there is also a bias from reading all texts at the same time, giving a frenetic and a sense of too much subjects at once.

ok all for me now got to go

**Reply by samuelhackwill** [reply]
hey jacob, thank you so much for reading all my digressions sauvages de sociopathe

your remarks made me realize that, i think, the goal of this sequence is to say something about inhospitality. So in a sense, the very short paragraph i wrote about racism is probably what i've been avoiding for the preceding 8 pages. I think inhospitality has to do with the global theme of inclusivity and how computers are causing one part of the population to be abandoned by society. 

i think this is the rationale for including this sequence in le tiret du six.
